build-release.sh

```diff
@@ -331,11 +331,11 @@ if [ "${build_mono}" == "1" ]; then
   cp out/macosx/tools-mono/godot.osx.opt.tools.universal.mono Godot_mono.app/Contents/MacOS/Godot
   mkdir -p Godot_mono.app/Contents/{Frameworks,Resources}
   mkdir -p Godot_mono.app/Contents/{Frameworks,Resources}/GodotSharp
-  mkdir -p Godot_mono.app/Contents/{Frameworks,Resources}/GodotSharp/Mono
+  mkdir -p Godot_mono.app/Contents/Resources/GodotSharp/Mono
   cp -rp out/macosx/tools-mono/GodotSharp/Api Godot_mono.app/Contents/Frameworks/GodotSharp
-  cp -rp out/macosx/tools-mono/GodotSharp/Mono/lib Godot_mono.app/Contents/Frameworks/GodotSharp/Mono
   cp -rp out/macosx/tools-mono/GodotSharp/Tools Godot_mono.app/Contents/Frameworks/GodotSharp
-  cp -rp out/macosx/tools-mono/GodotSharp/Mono/etc Godot_mono.app/Contents/Resources/GodotSharp/Mono
+  cp -rp out/macosx/tools-mono/GodotSharp/Mono Godot_mono.app/Contents/Resources/GodotSharp/Mono
```

Comment:
IINM this will end up as `Godot_mono.app/Contents/Resources/GodotSharp/Mono/Mono`, is that intended? If not, the `mkdir` should be removed and then the destination should be the `GodotSharp/` folder.

Reply:
Yes, my bad. That's a copy-paste casualty. (I don't have the means of reproducing the entire Godot build setup here, unfortunately, so having to kind of shunt my own local scripts into it.) Will fix. Thanks!